.github/hypercritic.yml

```diff
@@ -3,4 +3,6 @@ commitRegExp: ~
 commitRegExpExplanation: ~
 ignoreWords:
   - config
+  - vscode
```

Comment:
Unfortunately, bot fetches configuration from master branch and not the current one.